### ISSUE TITLE
Debug call stack "[ Immediate ]" text is too low contrast

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#86342]
One of its ancestors is using `opacity: 0.35` which ends up being approximately `#5F5F5F` which fails WCAG AA

![image](https://user-images.githubusercontent.com/2193314/70191311-32767a00-16ad-11ea-8220-985d7a3c753e.png)

![image](https://user-images.githubusercontent.com/2193314/70191327-3d310f00-16ad-11ea-93f2-4eeaa06b7828.png)

Light is bad too:

![image](https://user-images.githubusercontent.com/2193314/70191394-723d6180-16ad-11ea-9e95-5f4b3b4b4c52.png)

**Comment by Tyriar** [comment]
This is also causing the restart frame button to look disabled:

![image](https://user-images.githubusercontent.com/2193314/70191497-cb0cfa00-16ad-11ea-949f-5b035f6f2c82.png)

**Comment by isidorn** [comment]
Thanks for filling this.
Restart frame should not appear on those frames. I can look into that.
@misolori can you check what opacity passes the test when you find the time?

Nothing new, assigning to December

**Comment by Tyriar** [comment]
I think when I measured yesterday 0.6 worked and 0.5 just failed, at least for Dark+.

**Comment by misolori** [comment]
0.65 should give you the right contrast for dark:

<img width="50%" src="https://user-images.githubusercontent.com/35271042/70263781-fd6d3480-174b-11ea-948f-9f8c608d7f6e.png"/>

and 0.95 for light:

<img width="50%" src="https://user-images.githubusercontent.com/35271042/70264093-9dc35900-174c-11ea-921d-a60ca841ddc9.png"/>

**Comment by isidorn** [comment]
If I use the 0.95 opacity in light theme that completely defeats the purpose of making this not so much in the users face. MIght as well not use opacity then.
0.65 in dark could work.  0.65 works in light also.
For now going with 0.65 in both themes until somebody complains.

**Comment by misolori** [comment]
I agree that it doesn't look great on light, but 0.65 on light would still fail the color contrast ratio:

<img width="50%" src="https://user-images.githubusercontent.com/35271042/70545608-b489f700-1b22-11ea-82d2-1ac4d70206ea.png">

Re-opening since it doesn't fix the issue in Light.

**Comment by isidorn** [comment]
Light theme imho looks decent, thus closing for now.
Dark theme was a bigger problem
